docs/faq.rst

```diff
@@ -509,6 +509,31 @@ Please disclose security issues responsibly.
 Common issues
 #############
 
+Why does Borg extract hang after some time?
```

Comment:
i think this should be split into 2 sections:

1. avoiding hangs by configuring ssh/sshd alive interval/count for both server and client side. this makes sure from both sides that the connection gets terminated and does not just hang for a longer time.

2. how to deal with unstable connections (doing smaller ops, using rsync)